pulseapi/profiles/test_models.py

```diff
@@ -0,0 +1,19 @@
+"""UserProfile Model Generator"""
+from pulseapi.profiles.models import UserProfile
+
+import factory
+from faker import Factory
+
+fake = Factory.create()
+
+
+class UserProfileFactory(factory.Factory):
+    """Generate UserProfiles for tests"""
+    is_active = True
+    user_bio = factory.LazyAttribute(
+        lambda o: 'user_bio {}'.format(''.join(fake.text(max_nb_chars=130)))
```

Comment:
nit, but can we name this parameter? Like `'user_bio {fakeuser}'.format(fakeuser = ....)`?